### DISCUSSION TITLE
Proposte prossime versioni

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#699] [Idee e proposte]
Buonasera propongo le seguenti funzioni

1. Orario e planning settimanale per il docente
2. La funzione della scuola secondaria di secondo grado
3. La funzionalità da parte del coordinatore di segnalare nel registro le supplenze in modo che il supplente lo sappia in anticipo
4. La possibilità da parte del dirigente di omologare una nota disciplinare rendendola immodificabile e/o incancellabile
5. La possibilità di non chiedere la giustificazione ai ritardi
6. Il voto + e -
7. La possibilità che la famiglia richieda un attesta ( per es. un pon).
8. La possibilità di avere un account genitore/docente senza crearne 2

**Comment by Davide031** [comment]
anche di mettere una nota dalla finestra note senza dover cambiare schermate più volte

**Comment by trinko** [comment]
>     2. La funzione della scuola secondaria di secondo grado

Non ho capito cosa intendi.
 
>     3. La funzionalità da parte del coordinatore di segnalare nel registro le supplenze in modo che il supplente lo sappia in anticipo

Si può fare con gli avvisi. Unico limite è che è lo staff e non il coordinatore a inserirli, ma questo mi pare coerente con il fatto che il docente indicato per la supplenza non appartiene necessariamente al consiglio di classe del coordinatore.
 
>     5. La possibilità di non chiedere la giustificazione ai ritardi

Si può fare con l'impostazione del ritardo breve (la trovi nei PARAMETRI): basta indicare un numero di minuti adeguato. Es. con 120, chi entra in ritardo entro le prime 2 ore non giustifica.

>     7. La possibilità che la famiglia richieda un attesta ( per es. un pon).

Si può fare con i moduli. Unico limite è che il destinatario della richiesta non può essere la segreteria, ma lo staff.
